assignments/callbacks.js

```diff
@@ -24,29 +24,92 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+
+  return cb(arr.length);
 }
 
+getLength(items, function(length) {
+  console.log(length)
+});
+// answer ↑↑
+
+
+
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr.pop());
 }
 
+last(items, function(lastItem) {
+  console.log(lastItem)
+});
+// answer ↑↑
+
+
+
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
 
+sumNums(10, 20, function(result){
+  console.log(result)
+});
+// answer ↑↑
+
+
+
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y)
 }
 
+multiplyNums(2, 5, function(result) {
+  console.log(result)
+});
+// answer  ↑↑
+
+
+
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
-}
+  if (list.includes(item)) {
+    console.log(item);
+  } else {
+    console.log(`${item} is not in the list!`)
+  }
```

Comment:
technically this wants you to pass true or false to the callback rather than the item or the message, but i get why you're doing it this way!